jdbcSource/src/main/java/io/vantiq/extsrc/jdbcSource/JDBCCore.java

```diff
@@ -243,12 +244,19 @@ public void executePublish(ExtensionServiceMessage message) {
                 String queryString = (String) request.get("query");
                 int data = localJDBC.processPublish(queryString);
                 log.trace("The returned integer value from Publish Query is the following: ", data);
+            } else if (request.get("query") instanceof List) {
```

Comment:
can you check here for `instanceof List<String>` -- since a list of integers as it stands isn't gonna work well.

Reply:
I wanted to do this, but Java only lets me check for a generic list. I have a test that handles a list of integers, and I added a catch block for "ClassCastException" so at least we can have a better error message.

Let me know if that's alright.